pkg/stacker/build.go

```diff
@@ -820,6 +820,12 @@ func SetupLayerConfig(config types.StackerConfig, c *container.Container, l type
 	}
 
 	for _, bind := range l.Binds {
+		_, err := os.Stat(bind.Source)
+		// c.BindMount allows files to not exist and will create them, but not so for directories.
```

Comment:
is this true? BindMount() sets create=dir for dirs and create=file for files, but that's just for the destination right? If the source isn't there it doesn't matter what type it is, I thought.

anyway, the code is right otherwise. lgtm

Reply:
I can drop the comment; but it shows the general confusion;  the createOpts don't say if they're for the source or the dest; but thinking about it for a second, it's always for dest.

I might touch those variables...

Alternatively, I can move the check into the BindMount() call, which would protect other paths as well.

Thoughts?

Reply:
yeah probably a good idea to just do the check in bindmount() as you suggest. not as likely to catch, but why not...